### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by cblecker**
This adds issue templates to allow users to submit common kinds of requests.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/repo-create.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+e.g. cloud-provider-foo
+
+### Which Organization should it reside
+e.g. kubernetes-sigs
+
+### Who should be provided access to this repository
+e.g (at)example_user should have write access
+
+### Has sig-architecture reviewed/approved this request?
+e.g. Yes, and here's the link.
```
Comment (spiffxp) [inline .github/ISSUE_TEMPLATE/repo-create.md]: For kubernetes-sigs, sig-arch doesn't have a say, it's just the owning sig.

I am preferential to linkable proof in the form of a mailing list thread or meeting minutes.  I'm not going to accept a slack link.  Really don't know how I feel about a youtube link either.

File: .github/ISSUE_TEMPLATE/integrations.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Integrations or Webhooks
```
Comment (spiffxp) [inline .github/ISSUE_TEMPLATE/integrations.md]: Nit: Maybe include GitHub Apps in the wording somewhere in here too

**Comment by spiffxp** [comment]
/hold
These are awesome.  My main reservation is on the repo-create form.  The rules are different depending on which org you want to create in.  I don't know if we have to go so far as to state which repos people can choose from (ie: disallow creating in kubernetes-incubator).

**Comment by cblecker** [comment]
@spiffxp Addressed comments. PTAL

**Comment by spiffxp** [comment]
/approve
I'll leave it to someone else to /lgtm

**Comment by cblecker** [comment]
/hold cancel

If there are other comments, open to iterating.

**Pull request merged**